src/gui/inputdevices/InputEvents.cpp

```diff
@@ -97,48 +72,48 @@ auto InputEvents::translateDeviceType(GdkDevice* device, Settings* settings) ->
     return translateDeviceType(gdk_device_get_name(device), gdk_device_get_source(device), settings);
 }
 
-auto InputEvents::translateEvent(GdkEvent* sourceEvent, Settings* settings) -> InputEvent* {
-    auto targetEvent = new InputEvent();
+auto InputEvents::translateEvent(GdkEvent* sourceEvent, Settings* settings) -> InputEvent {
+    InputEvent targetEvent{};
 
-    targetEvent->sourceEvent = sourceEvent;
+    targetEvent.sourceEvent = sourceEvent;
 
     // Map the event type to our internal ones
     GdkEventType sourceEventType = gdk_event_get_event_type(sourceEvent);
-    targetEvent->type = translateEventType(sourceEventType);
+    targetEvent.type = translateEventType(sourceEventType);
 
     GdkDevice* device = gdk_event_get_source_device(sourceEvent);
-    targetEvent->deviceClass = translateDeviceType(device, settings);
+    targetEvent.deviceClass = translateDeviceType(device, settings);
 
-    targetEvent->deviceName = const_cast<gchar*>(gdk_device_get_name(device));
+    targetEvent.deviceName = const_cast<gchar*>(gdk_device_get_name(device));
 
     // Copy both coordinates of the event
-    gdk_event_get_root_coords(sourceEvent, &(targetEvent->absoluteX), &(targetEvent->absoluteY));
-    gdk_event_get_coords(sourceEvent, &(targetEvent->relativeX), &(targetEvent->relativeY));
+    gdk_event_get_root_coords(sourceEvent, &(targetEvent.absoluteX), &(targetEvent.absoluteY));
+    gdk_event_get_coords(sourceEvent, &(targetEvent.relativeX), &(targetEvent.relativeY));
```

Comment:
Very minor, but the `&(targetEvent.XYZ)` style is not consistent with the `&targetEvent.pressure` below.